pages/blog.js

```diff
@@ -28,7 +43,33 @@ class Blog extends React.Component {
 
     return { posts: json };
   }
+
+  getGategoriesList(url) {
+    const { posts } = this.props;
+    const chosenCategory = url.query.chosen;
+    const categories = posts
+      .map(post => post.tags)
```

Comment:
"Categories" or "Tags"? Let's use consistent names for posts porperties.

Reply:
Done.